src/components/views/messages/MImageBody.js

```diff
@@ -321,13 +338,17 @@ export default class MImageBody extends React.Component {
             // By doing this, the image "pops" into the timeline, but is still restricted
             // by the same width and height logic below.
             if (!this.state.loadedImageDimensions) {
-                return this.wrapImage(contentUrl,
+                let imageElement = (
                     <img style={{display: 'none'}} src={thumbUrl} ref="image"
-                        alt={content.body}
-                        onError={this.onImageError}
-                        onLoad={this.onImageLoad}
-                    />,
+                         alt={content.body}
+                         onError={this.onImageError}
+                         onLoad={this.onImageLoad}
+                    />
                 );
+                if (!this.state.showImage) {
```

Comment:
nit: if/else here?

Reply:
There's not really an else condition, though could be done.